### PR TITLE
Forward to list on missing object during deletion

### PR DESCRIPTION
**Pull request by bobvandevijver**
When a object is not found during deletion (for example, when clicking the confirm button twice on the confirm page), you get an Exception which is too general: Object could not be removed. 

This PR introduces an automatic forward, which creates a notification that it could not be removed as it already was removed. 

Also added some tweaks in parts of the code to make it more consistent.

### CODE REVIEW

**Comment by tobias-93** [comment]
I'm :+1: for this, an Exception is too harsh in this case.

File: Resources/translations/Admingenerator.fr.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@ action.object.delete:
     confirm:  Etes-vous sur de vouloir supprimer cet élement?
     success:  L'élément sélectionné a été supprimé.
     error:    L'élément sélectionné ne peut pas être supprimé.
+    notfound: Element ne peut pas être supprimé car il est déjà supprimé.
```
Comment (sescandell) [inline Resources/translations/Admingenerator.fr.yml]: French one: *Suppression impossible : l'élément est inconnu*

File: Resources/templates/Doctrine/ActionsBuilderAction.php.twig [hunk 2]
```diff
@@ -12,7 +12,7 @@
         ${{ builder.ModelClass }} = $this->getObjectQuery($pk)->getOneOrNullResult();
 
         if (!${{ builder.ModelClass }}) {
-            throw new \InvalidArgumentException("No {{ model }} found on {{ builder.getFieldGuesser().getModelPrimaryKeyName(model) }} : $pk");
+            throw new NotFoundHttpException("No {{ model }} found on {{ builder.getFieldGuesser().getModelPrimaryKeyName(model) }} : $pk");
```
Comment (sescandell) [inline Resources/templates/Doctrine/ActionsBuilderAction.php.twig]: Take care, this is a BC break that need a comment in the UPDATE doc.

**Comment by sescandell** [comment]
2 comments, otherwise : :+1:

File: Resources/templates/Doctrine/ActionsBuilderAction.php.twig (hunk 2 above)
Comment (bobvandevijver) [inline Resources/templates/Doctrine/ActionsBuilderAction.php.twig]: Not really, as by default it is catched. Only when you use the new `throw_exception` parameter the thrown exception changes. Nevertheless, I will add a note.

File: Resources/translations/Admingenerator.fr.yml (hunk 1 above)
Comment (bobvandevijver) [inline Resources/translations/Admingenerator.fr.yml]: Done.

**Comment by sescandell** [comment]
:+1: good for me

**Pull request merged**